index.html

```diff
@@ -260,22 +273,71 @@ <h2 id="total-contributors" style="text-align: center; font-family: 'Agency FB',
     </div>
 
 
-
-
-
-
-
-
-
     <footer class="page-footer font-small unique-color-dark pt-4">
-      <div class="container1">
+      <div class="container" style="margin-left: 50px;">
         <div class="list-unstyled list-inline text-center py-2 text-black">
-          <br>
-          <br>
+          <br><br>
+    
           <!-- Created div Tag with Proper Styling for Footer Text aligned to Center just above Social Links -->
-          <div class="socials-text" classstyle="text-align: center; margin-bottom:50px">
+          <div class="socials-text" style="text-align: center; margin-bottom: 50px;">
+            <div class="footer-links" style="display: flex; justify-content: space-around; margin-bottom: 30px;">
+              <!-- Quick Links Section -->
+              <div class="quick-links">
+                <h3>Quick Links</h3>
+                <ul>
+                  <li><a href="#about">About Us</a></li>
+                  <li><a href="#contact">Contact</a></li>
+                  <li><a href="#contribute">Contribute Here</a></li>
+                </ul>
+              </div>
+    
+              <!-- Subscribe to Newsletter Section -->
+              <div class="newsletter">

```

Comment:
this is not fully functional right its only frontend

Reply:
sorry @ishita-43 I didn't get you

Reply:
how the entries enter by the user eg email in newsletter will be reached to the admin

Reply:
@ishita-43 Yes I only did the frontend intentionally. While creating the issue I mentioned that I will design the footer part.

Reply:
okay

Reply:
@ishita-43 won't this PR get merged. Its been days

Reply:
working on this!